src/NUnitFramework/testdata/PropertyAttributeTests.cs

```diff
@@ -37,6 +37,9 @@ public class FixtureWithProperties
 
 		[Test, Priority(5)]
 		public void Test3() { }
+
+        [Test, CustomProperty]
+        public void Tes4() { }
```

Comment:
There is a spelling error in this method name. The formatting of this diff looks wrong, but it is just because the rest of the file uses tabs instead of spaces, so it is fine.